eng/native/build-commons.sh

```diff
@@ -101,50 +101,7 @@ build_native()
         buildTool="make"
     fi
 
-    runtimeVersionHeaderFile="$intermediatesDir/../runtime_version.h"
     if [[ "$__SkipConfigure" == 0 ]]; then
-        # if msbuild is not supported, then set __SkipGenerateVersion to 1
```

Comment:
Shall we add the new 'copy default version' functionality in this common place, so we can continue to build `clr+libs.native+host.native` subsets without msbuild?

```sh
# currently for new platform bringup (where msbuild is not yet available), we use this script

# coreclr
/runtime/src/coreclr/build-runtime.sh -skipgenerateversion -skipmanaged -nopgooptimize \
    -os $OSName -cross -$OSArch

# libraries/native
/runtime/src/libraries/Native/build-native.sh -skipgenerateversion -skipmanaged -os $OSName \
    -cross -$OSArch

# corehost
/runtime/src/native/corehost/build.sh -skipmanaged -skipgenerateversion -os $OSName -cross -$OSArch \
    -policyver 0.0.1-dev -fxrver 0.0.1-dev -hostver 0.0.1-dev -apphostver 0.0.1-dev \
    -commithash $(git -C /runtime rev-parse HEAD)
```

Reply:
This is now called in a common place for all three build scripts.